test/ts/unit/collateralizer/collateralizer.ts

```diff
@@ -172,6 +176,61 @@ contract("CollateralizedContract (Unit Tests)", async (ACCOUNTS) => {
         });
     });
 
+    describe("Permissioning", () => {
+        describe("#addAuthorizedCollateralizeAgent", () => {
+            let txHash: string;
+
+            before(async () => {
+                txHash = await collateralizerContract.addAuthorizedCollateralizeAgent.sendTransactionAsync(
+                    AGENT,
+                );
+            });
+
+            it("adds specified agent to set of authorized agents", async () => {
+                const authorizedAgents = await collateralizerContract.getAuthorizedCollateralizeAgents.callAsync();
+                expect(authorizedAgents.includes(AGENT)).to.be.true;
+            });
```

Comment:
Please add a test for ensuring this is not a method that can be called by non-contract owner.